public_html/assets/css/responsive.css

```diff
@@ -31,6 +40,10 @@
         font-size: 50px;
         line-height: 65px;
     }
+
+    .cbailylaw__hero-img img {
+        bottom: -30%;
```

Comment:
I believe that this is pushing the image off of the bottom border of the hero section.

![Screenshot 2023-07-07 at 19-25-40 C Baily Law LLC](https://github.com/stratofax/cbailylaw/assets/37887507/685be48d-3ec0-4e71-812b-4746eef027f5)

Notice the gap between the section border and the base of the image.